app/services/check_answers/section_id_finder.rb

```diff
@@ -10,12 +10,14 @@ class SectionIdFinder
       dependant_details: "section-dependants-header",
       employment_status: "subsection-client_employment-header",
       employment: "subsection-client_pay-header",
+      income: "subsection-client_employment_income-header",
       benefits: "subsection-benefits-header",
       benefit_details: "subsection-benefits-header",
       other_income: "subsection-other_income-header",
       partner_details: "section-partner_details-header",
       partner_employment_status: "subsection-partner_employment-header",
       partner_employment: "subsection-partner_pay-header",
+      partner_income: "subsection-partner_pay-header",
```

Comment:
Yeah, this is slightly pre-empting the next PR. When there is a partner_income step it will point to the partner pay header, which will also exist by then. But before that point, neither exists, so this step is irrelevant.